### DISCUSSION TITLE
0.7.2

### DISCUSSION THREAD

**Discussion by kubilus1** [kubilus1/autoortho#482] [Announcements]
## What's Changed
* Lock multi-thread seek and read for same FH. by @kubilus1 in https://github.com/kubilus1/autoortho/pull/479

**Full Changelog**: https://github.com/kubilus1/autoortho/compare/0.7.1...0.7.2

<hr /><em>This discussion was created from the release <a href='https://github.com/kubilus1/autoortho/releases/tag/0.7.2'>0.7.2</a>.</em>

**Comment by block62** [comment]
Does anyone know where I can find the config to change something that makes Autoortho redownload the whole scenery package every time the download is interrupted? I forgot

**Comment by IkaroChagas** [comment]
After the update my PC is giving a blue screen with the watchdog violation error

**Comment by sergiolamares** [comment]
Anyone knows witch mod / plugin makes roads to be transparent? thanks

**Comment by fascox** [comment]
> Anyone knows witch mod / plugin makes roads to be transparent? thanks

try xroads

**Comment by garrybank** [comment]
As of today, 8th May 2024, Malware Bytes is identifying autoortho.exe as Malware ie Malware.AI.2741571920

I have tried "allowing" it but it has just stopped working. Pity as I think it is a great addition to X-Plane

**Comment by airbusseventhreeseven** [comment]
Much better than it originally was, good job. With the latest version I do still have a few crashes but it's doable

**Comment by axel248** [comment]
Hello,

Last update was back in January almost a year. Any news on updating the current version?

Thank you.
